api/integrations/sentry/apps.py

```diff
@@ -12,9 +12,17 @@ def ready(self):
             sentry_sdk.init(
                 dsn=settings.SENTRY_SDK_DSN,
                 integrations=[DjangoIntegration()],
-                traces_sample_rate=settings.SENTRY_TRACE_SAMPLE_RATE,
                 environment=settings.ENV,
+                traces_sampler=traces_sampler,
                 # If you wish to associate users to errors (assuming you are using
                 # django.contrib.auth) you may enable sending PII data.
                 send_default_pii=True,
             )
+
+
+def traces_sampler(ctx):
```

Comment:
I'm not a massive fan of this being defined in the `apps` module. Could we move it somewhere more appropriate? `samplers` or something?